benches/fixtures/json_example.json

```diff
@@ -0,0 +1,1217 @@
+[
+    {
+        "_id": "6623c00aa16be249c931db66",
+        "index": 0,
+        "guid": "67b63043-00ae-4dfc-9082-f1a994af30f3",
```

Comment:
What is the provenance of this example file? Just to make sure we don't run into any license issues with it.

Reply:
I am unable to find the source of the data as it was a few weeks ago, I took it from some example JSON site as I thought it looked quite realistic to how people use JSON. I'm unaware if there were any licenses on the data.